modules/core/include/opencv2/core/cuda.hpp

```diff
@@ -134,8 +134,8 @@ class CV_EXPORTS_W GpuMat
     CV_WRAP GpuMat(const GpuMat& m);
 
     //! constructor for GpuMat headers pointing to user-allocated data
-    CV_WRAP GpuMat(int rows, int cols, int type, void* data, size_t step = Mat::AUTO_STEP);
-    CV_WRAP GpuMat(Size size, int type, void* data, size_t step = Mat::AUTO_STEP);
+    GpuMat(int rows, int cols, int type, void* data, size_t step = Mat::AUTO_STEP);
+    GpuMat(Size size, int type, void* data, size_t step = Mat::AUTO_STEP);
```

Comment:
/cc @cv3d

Reply:
It is shame I caused such mess! Thanks for the cleanup.
BTW: It makes sense to disable args, while taking in consideration that we need to return `void` in order to support UMat wrapper.